test/browser_test.rb

```diff
@@ -65,6 +65,15 @@ class BrowserTest < Minitest::Test
     assert_equal "4", @browser.version
   end
 
+  test "detects chrome browser" do
+    @browser.ua = $ua["CHROME"]
```

Comment:
any update on this? not sure what should be changed here to have this PR accepted